test/RotationTest.cpp

```diff
@@ -26,7 +26,7 @@ namespace etrobocon2022_test {
     double rightMotorCount;
     double targetDistance;
     double tread = 125.0;
-    double error = (M_PI * tread / 360) * 2;  //誤差2度
+    double error = (M_PI * tread / 360) * 3;  //誤差3度
     int angle, pwm;
 
     // 90度左回頭の回頭誤差が１度未満かテスト
```

Comment:
```suggestion
    // 90度左回頭の回頭誤差が3度未満かテスト
```

それと、できれば許容する誤差の説明が欲しいです